### PR TITLE
[FIX] l10n_lu tags csv

### PR DESCRIPTION
**Pull request by sbidoul**
Some cleanup of l10n_lu tax tags csv, errors introduced in #19102 

Misplaced newlines, mainly.

@smetl @fgi-odoo 

cc/ @xavieralt @adrienpeiffer @benwillig 

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by smetl** [comment]
Your branch should target 9.0

**Comment by sbidoul** [comment]
If you don't mind I'll let you do that when merging. I'm testing on 10.0 right now.

**Comment by sbidoul** [comment]
I added a commit to fix an issue in the tax templates (two records with same id).

File: addons/l10n_lu/data/account.tax.template-2015.csv [hunk 1]
```diff
@@ -405,4 +405,6 @@ lu_2011_tax_VP-PA-3,Vente Prestations 3% - Pays,VP-PA-3,sale,percent,3,lu_2011_a
 lu_2011_tax_VP-PA-6,Vente Prestations 6% - Pays,VP-PA-6,sale,percent,6,lu_2011_account_4614111,lu_2011_account_4614111,lu_2011_chart_1,504,,"tag_lu_200,tag_lu_278"
 lu_2015_tax_VP-PA-8,Vente Prestations 8% - Pays,VP-PA-8,sale,percent,8,lu_2011_account_4614111,lu_2011_account_4614111,lu_2011_chart_1,505,,"tag_lu_294,tag_lu_218"
 lu_2015_tax_SANS,Sans Taxes,SANS,purchase,percent,0,,,lu_2011_chart_1,506,,tag_lu_598
-lu_2015_tax_SANS,Sans Taxes,SANS,sale,percent,0,,,lu_2011_chart_1,507,,tag_lu_598
\ No newline at end of file
+lu_2015_tax_SANS,Sans Taxes,SANS,sale,percent,0,,,lu_2011_chart_1,507,,tag_lu_598
```
Comment (mart-e) [inline addons/l10n_lu/data/account.tax.template-2015.csv]: Well actually you don't fix the bug as there is still twice the same row.  If they have the same id, the install will update the first line with the second one. So, in the end, we get only the sale line.
I agree we should keep it for backward compatibility (even if a tax template is probably not of much use) but you can remove the purchase line then.

Comment (sbidoul) [reply addons/l10n_lu/data/account.tax.template-2015.csv]: For this specific case I don't think backward compatibility is really important. So I propose to remove the bad one.

**Comment by sbidoul** [comment]
Closing in favor of the 9.0 branch: https://github.com/odoo/odoo/pull/20678

**Pull request closed without merging**